lib/bolt/transport/winrm/connection.rb

```diff
@@ -116,12 +117,15 @@ def execute(command)
             # propagate to the main thread via the shell, there's no chance
             # they will be unhandled, so the default stack trace is unneeded.
             Thread.current.report_on_exception = false
-            result = @session.run(command)
-            out_wr << result.stdout
-            err_wr << result.stderr
-            out_wr.close
-            err_wr.close
-            result.exitcode
+
+            @connection.shell(:powershell) do |session|
+              result = session.run(command)
+              out_wr << result.stdout
+              err_wr << result.stderr
+              out_wr.close
+              err_wr.close
```

Comment:
Just checking, but we should be able to remove these two lines due to the `ensure` below?